### ISSUE TITLE
Android Build error with nativewind and expo-updates

### ISSUE THREAD

**Issue by kelvinpompey** [nativewind/nativewind#1158]
**Describe the bug**
After adding expo-updates@0.25.27 to an expo project with nativewind@4.1.18 the project fails to build on Android. It fails with the following error: 

```
[RUN_GRADLEW] Error: @build-script-error-begin
[RUN_GRADLEW] Error loading assets JSON from Metro. Ensure you've followed all expo-updates installation steps correctly. SHA-1 for file /private/var/folders/z4/f8k53rr971n_dyn54z4vptmc0000gn/T/eas-build-local-nodejs/3019dc79-e8ef-48b7-81eb-e091699b67b3/build/node_modules/react-native-css-interop/.cache/android.js is not computed.
[RUN_GRADLEW]          Potential causes:
[RUN_GRADLEW]            1) You have symlinks in your project - watchman does not follow symlinks.
[RUN_GRADLEW]            2) Check `blockList` in your metro.config.js and make sure it isn't excluding the file path.
[RUN_GRADLEW] @build-script-error-end
[RUN_GRADLEW]     at createManifestForBuildAsync (/private/var/folders/z4/f8k53rr971n_dyn54z4vptmc0000gn/T/eas-build-local-nodejs/3019dc79-e8ef-48b7-81eb-e091699b67b3/build/node_modules/expo-updates/utils/build/createManifestForBuildAsync.js:44:15)
[RUN_GRADLEW]     at processTicksAndRejections (node:internal/process/task_queues:95:5)
[RUN_GRADLEW]     at async Promise.all (index 0)
[RUN_GRADLEW]     at /private/var/folders/z4/f8k53rr971n_dyn54z4vptmc0000gn/T/eas-build-local-nodejs/3019dc79-e8ef-48b7-81eb-e091699b67b3/build/node_modules/expo-updates/utils/build/createUpdatesResources.js:26:5
[RUN_GRADLEW] FAILURE: Build failed with an exception.
[RUN_GRADLEW] * What went wrong:
[RUN_GRADLEW] Execution failed for task ':app:createReleaseUpdatesResources'.
[RUN_GRADLEW] > Process 'command 'node'' finished with non-zero exit value 1
[RUN_GRADLEW] * Try:
[RUN_GRADLEW] > Run with --stacktrace option to get the stack trace.
[RUN_GRADLEW] > Run with --info or --debug option to get more log output.
[RUN_GRADLEW] > Run with --scan to get full insights.
[RUN_GRADLEW] > Get more help at https://help.gradle.org.
[RUN_GRADLEW] BUILD FAILED in 2m 39s
```

**Reproduction**
A project was generated with 
  ```npx create-expo-stack@latest --nativewind --expo-router```
  
  I then installed expo-updates with 
  
  ```npx expo install expo-updates```
  
  I then tried to build with eas
  ```
  eas build -p android --profile preview --local
```

I am using Java 17 
OpenJDK Runtime Environment Zulu17.52+17-CA (build 17.0.12+7-LTS)

A repo demonstrating the issue is available: 

https://github.com/kelvinpompey/nativewind-expo-updates-issue

**Comment by marklawlor** [comment]
Fixed in `4.1.19`

**Comment by MrPiao** [comment]
I am running into this exact same issue – except on iOS.

> [RUN_FASTLANE] ▸ ** ARCHIVE FAILED **
[RUN_FASTLANE] ▸ The following build commands failed:
[RUN_FASTLANE] ▸ 	PhaseScriptExecution [CP-User]\ Generate\ updates\ resources\ for\ expo-updates /Users/foo/Library/Developer/Xcode/DerivedData/bar-ciwftnmozkgpdsdzjbnlnyhimvaq/Build/Intermediates.noindex/ArchiveIntermediates/bonfireapp/IntermediateBuildFilesPath/Pods.build/Release-iphoneos/EXUpdates.build/Script-46EB2E0005B870.sh (in target 'EXUpdates' from project 'Pods')

> ❌  error: SHA-1 for file /private/var/folders/6n/z_kx6kxj30j5tpbr0tjxs0gw0000gn/T/eas-build-local-nodejs/a93826f1-c6b2-4df0-900b-56e40a207152/build/apps/expo/node_modules/react-native-css-interop/.cache/ios.js is not computed.

Running `nativewind@4.1.20` and `expo-updates@0.25.27`. I confirmed that the issue goes away if I remove `expo-updates` from the project.

**Comment by GMoon777** [comment]
Same problem here

NativeWind v4.1.20
Expo updates v0.25.27

```
❌  error: SHA-1 for file /Users/expo/workingdir/build/node_modules/react-native-css-interop/.cache/ios.js (/Users/expo/workingdir/build/node_modules/react-native-css-interop/.cache/ios.js) is not computed.

❌ Metro encountered an error:
SHA-1 for file /Users/expo/workingdir/build/node_modules/react-native-css-interop/.cache/ios.js (/Users/expo/workingdir/build/node_modules/react-native-css-interop/.cache/ios.js) is not computed.
         Potential causes:
           1) You have symlinks in your project - watchman does not follow symlinks.
           2) Check `blockList` in your metro.config.js and make sure it isn't excluding the file path.
```

**Comment by TCGXR** [comment]
Same here, any idea how to fix it?

**Comment by MrPiao** [comment]
See #1162 
TL;DR: .21 should fix the issue. It did for me.

**Comment by TCGXR** [comment]
Looks like it is not fixed an Android - even with .21. I have exactly the same error but on android.

"nativewind": "^4.1.21",
"tailwindcss": "^3.4.14"

    Task :app:createBundleReleaseJsAndAssets
    debug Reading Metro config from C:\Dev\tcgxr_app\react_app\metro.config.js
    warning: the transform cache was reset.
    Welcome to Metro v0.80.12
    Fast - Scalable - Integrated

error SHA-1 for file C:\Dev\tcgxr_app\react_app\node_modules\react-native-css-interop.cache/android.js is not computed.
Potential causes:
1) You have symlinks in your project - watchman does not follow symlinks.
2) Check blockList in your metro.config.js and make sure it isn't excluding the file path.
ReferenceError: SHA-1 for file C:\Dev\tcgxr_app\react_app\node_modules\react-native-css-interop.cache/android.js is not computed.
Potential causes:
1) You have symlinks in your project - watchman does not follow symlinks.
2) Check blockList in your metro.config.js and make sure it isn't excluding the file path.
at DependencyGraph.getSha1 (C:\Dev\tcgxr_app\react_app\node_modules\metro\src\node-haste\DependencyGraph.js:179:13)
at Transformer._getSha1 (C:\Dev\tcgxr_app\react_app\node_modules\metro\src\Bundler.js:15:26)
at Transformer.transformFile (C:\Dev\tcgxr_app\react_app\node_modules\metro\src\DeltaBundler\Transformer.js:88:19)
at Bundler.transformFile (C:\Dev\tcgxr_app\react_app\node_modules\metro\src\Bundler.js:43:30)
at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

**Comment by marklawlor** [comment]
@TCGXR Please keep your issue within the same issue you opened. Commenting on multiple closed issues generates noise for all contributors and is a good way to annoy the people trying to help you.
